### ISSUE TITLE
Mac intel host only bots post-submit queue time > SLO (10mins)

### ISSUE THREAD

**Issue by keyonghan** [flutter/flutter#130837]
For past two weeks 07/03-07/16, the post-submit queue time (90th%) of Mac Intel host only bots is 11.30 mins, which exceeds SLO 10mins.

**Comment by keyonghan** [comment]
When multiple engine commits land in a short time, they will consume all Mac intel bots with normal priority.
Mac intel host only targets from framework with lower priority (scheduled via back filler) will be scheduled with increased queue time.

Related change: https://github.com/flutter/engine/pull/43788 to increase orchestrator timeout to unblock engine tree. This may potentially make the framework tree queue time higher.

https://github.com/flutter/flutter/issues/130834 is created to throttle commit merges, and is expected to improve the queue time.

**Comment by keyonghan** [comment]
Compared to prod pool, the try pool queue time is 1.35min with a SLO of 5 mins. 
From capacity side: prod has [28 bots](https://chromium-swarm.appspot.com/botlist?c=id&c=task&c=os&c=status&d=asc&f=pool%3Aluci.flutter.prod&f=os%3AMac-12&f=device_type%3Anone&f=gpu%3A8086&k=gpu&s=id) whereas try has [75 bots](https://chromium-swarm.appspot.com/botlist?c=id&c=task&c=cores&c=os&c=status&d=asc&f=os%3AMac-12&f=device_type%3Anone&f=gpu%3A8086&f=pool%3Aluci.flutter.try&k=pool&s=id)

We should relocate some capacity from try to prod. Considering linux vms (prod: 160 v.s. try: 275, roughly 2/3), we can try moving 12 bots from try to prod, with (prod: 40, try: 63). @yusuf-goog Could you help with this change?

**Comment by keyonghan** [comment]
https://chrome-internal-review.googlesource.com/c/infradata/config/+/6183955 moved 12 bots from try to prod, and the recent cycle queue time drops to 1.97 mins. Closing.
